### PR TITLE
add policy support

### PR DESCRIPTION
**Pull request by zreigz**

### CODE REVIEW

File: resources/policy/bundles/asm-policy-v0.0.1/1.4.1_asm-request-authn-prohibited-output-headers.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+apiVersion: constraints.gatekeeper.sh/v1beta1
+kind: AsmRequestAuthnProhibitedOutputHeaders
+metadata:
+  name: asm-policy-v0.0.1-asm-request-authn-prohibited-output-headers
+  labels:
+    policycontroller.gke.io/bundleName: asm-policy-v0.0.1
```
Comment (michaeljguarino) [inline resources/policy/bundles/asm-policy-v0.0.1/1.4.1_asm-request-authn-prohibited-output-headers.yaml:19]: I do think we should potentially rename the scope of these annotations from `policycontroller.gke.io` to `policy.plural.sh`

File: resources/policy/bundles/asm-policy-v0.0.1/LICENSE [hunk 2]
```diff
@@ -0,0 +1,202 @@
+
```
Comment (michaeljguarino) [inline resources/policy/bundles/asm-policy-v0.0.1/LICENSE:1]: we can remove this file i think

File: resources/policy/bundles/asm-policy-v0.0.1/README.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
+# Anthos Service Mesh Policy v0.0.1
```
Comment (michaeljguarino) [inline resources/policy/bundles/asm-policy-v0.0.1/README.md:1]: Let's clear this readme and write a fresh one (doesn't have to be in this PR though)

File: resources/policy/bundles/asm-policy-v0.0.1/1.4.1_asm-request-authn-prohibited-output-headers.yaml (hunk 1 above)
Comment (zreigz) [reply resources/policy/bundles/asm-policy-v0.0.1/1.4.1_asm-request-authn-prohibited-output-headers.yaml]: done

File: resources/policy/bundles/asm-policy-v0.0.1/LICENSE (hunk 2 above)
Comment (zreigz) [reply resources/policy/bundles/asm-policy-v0.0.1/LICENSE]: done

File: resources/policy/bundles/asm-policy-v0.0.1/README.md (hunk 3 above)
Comment (zreigz) [reply resources/policy/bundles/asm-policy-v0.0.1/README.md:1]: done

**Review by michaeljguarino — APPROVED** [review]

**Pull request merged**